creator-node/src/services/prometheusMonitoring/prometheusRegistry.js

```diff
@@ -0,0 +1,43 @@
+const promClient = require('prom-client')
+
+module.exports = class PrometheusRegistry {
+  init({ collectDefaultMetrics = true }) {
+    this.register = new promClient.Registry()
+
+    // Add a default label which is added to all metrics
+    this.register.setDefaultLabels({
+      serviceType: 'audius_content_node'
+    })
+
+    this.setupDefaultMetrics(collectDefaultMetrics)
+  }
+
+  getAllMetricData() {
+    return this.register.metrics()
+  }
+
+  async getMetricInstance(name) {
+    return this.register.getSingleMetric(name)
```

Comment:
It seems like we're still doing something similar to [my PR](https://github.com/AudiusProject/audius-protocol/blob/19acdcd28315a808a95d4e5337e670a803fe5998/creator-node/src/services/PrometheusMetric.js#L31-L72) except instead of using our own `collection` object, we're using the prom-client registry internals.

While super trivial, we do miss out on [this small bit of UX](https://github.com/AudiusProject/audius-protocol/blob/19acdcd28315a808a95d4e5337e670a803fe5998/creator-node/src/services/PrometheusMetric.js#L114-L118) that eliminates us having to remember if we're using a metric that supports `.set()` vs `.observe()`.

Reply:
i do think we should have explicit API's that the team can use for gauge vs histogram and they should pick which one to use

Reply:
Sure thing, I'm always teetering between wrapping APIs and exposing them almost fully, so I definitely understand the value in both. I'm 100% down with whatever you choose! And like you said, we can find emergent patterns over time.

Yeah, the set/observe difference is silly.

Major +1 on the caller knowing the metric type. We definitely should not hide the gauge/histogram/etc types. Plus, I can't think of a way we would even be able to hide it.

+1 on it having the added benefit of building shared knowledge.

Reply:
sweet! gonna leave unwrapped for now, lets revisit later